src/Microsoft.FeatureManagement.AspNetCore/Microsoft.FeatureManagement.AspNetCore.csproj

```diff
@@ -17,15 +17,19 @@
     <PackageIconUrl>https://aka.ms/AzureAppConfigurationPackageIcon</PackageIconUrl>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
   </PropertyGroup>
-  
+
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.FeatureManagement\Microsoft.FeatureManagement.csproj" />
   </ItemGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.TagHelpers" Version="2.1.3" />
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'" >
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.TagHelpers" Version="2.2.0" />
```

Comment:
We need to stick with 2.1.x versions even here for full supported integration with ASP.NET Core 2.1 apps. 2.1.3 should be fine since it is the latest 2.1.x patch version.

.NET Core 2.1 [is still supported](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).

Reply:
Sorry skipped this one, version reverted to the previous value

Reply:
Are you okay with keeping package updates for test projects?